src/cache/cache_las.c

```diff
@@ -424,22 +424,23 @@ __wt_las_page_skip_locked(WT_SESSION_IMPL *session, WT_REF *ref)
 	/*
 	 * Skip lookaside pages if reading as of a timestamp, we evicted new
 	 * versions of data and all the updates are in the past.
-	 */
-	if (ref->page_las->skew_newest &&
-	    txn->read_timestamp > ref->page_las->unstable_durable_timestamp)
-		return (true);
```

Comment:
This is the main fix here: include the timestamp from prepapred updates in the calculated `unstable_timestamp`.